src/Libraries/RevitNodes/Elements/Views/ScheduleView.cs

```diff
@@ -156,34 +171,87 @@ public static ScheduleView CreateSchedule(Category category, string name, string
         /// <summary>
         ///     Remove Schedule Field from Schedule.
         /// </summary>
-        /// <param name="field"></param>
+        /// <param name="fields"></param>
         /// <returns name="View">View Schedule</returns>
-        public ScheduleView RemoveField(Revit.Schedules.ScheduleField field)
+        public ScheduleView RemoveField(List<Revit.Schedules.ScheduleField> fields)
         {
-            if (this.Fields.Any(x => x.Name == field.Name))
+            var doc = DocumentManager.Instance.CurrentDBDocument;
+            TransactionManager.Instance.EnsureInTransaction(doc);
+
+            foreach (Revit.Schedules.ScheduleField field in fields)
             {
-                var doc = DocumentManager.Instance.CurrentDBDocument;
-                TransactionManager.Instance.EnsureInTransaction(doc);
-                this.InternalViewSchedule.Definition.RemoveField(field.InternalScheduleField.FieldId);
-                TransactionManager.Instance.TransactionTaskDone();
+                if (this.Fields.Any(x => x.Name == field.Name))
+                {
+                    this.InternalViewSchedule.Definition.RemoveField(field.InternalScheduleField.FieldId);
+                }
             }
 
+            TransactionManager.Instance.TransactionTaskDone();
+
             return this;
         }
 
         /// <summary>
         ///     Add Field.
         /// </summary>
-        /// <param name="field"></param>
+        /// <param name="fields"></param>
         /// <returns></returns>
-        public ScheduleView AddField(Revit.Schedules.SchedulableField field)
+        public ScheduleView AddFields(List<Revit.Schedules.SchedulableField> fields)
         {
-            if (!this.Fields.Any(x => x.Name == field.Name))
+            var doc = DocumentManager.Instance.CurrentDBDocument;
+            TransactionManager.Instance.EnsureInTransaction(doc);
+
+            foreach (Revit.Schedules.SchedulableField field in fields)
+            {
+                if (!this.Fields.Any(x => x.Name == field.Name))
+                {
+                    this.InternalViewSchedule.Definition.AddField(field.InternalSchedulableField);
+                }
+            }
+
+            TransactionManager.Instance.TransactionTaskDone();
+
+            return this;
+        }
+
+        /// <summary>
+        ///     Export View Schedule to CSV, TSV etc.
+        /// </summary>
+        /// <param name="path">A valid file path with file extension.</param>
+        /// <param name="exportOptions">Export Options. If null, default will be used.</param>
+        /// <returns></returns>
+        public ScheduleView Export(
+            string path, 
+            [DefaultArgumentAttribute("null")] ScheduleExportOptions exportOptions)
```

Comment:
I really don't like this use of null, something we did with @moethu's prs where he needed similar functionality was to define a static constructor for a default token:

https://github.com/DynamoDS/DynamoRevit/blob/7270187e8688fbd8ec538b4617810dccdb9a695f/src/Libraries/RevitNodes/Elements/TextNoteType.cs#L106

then create one of those as your optional argument:
https://github.com/DynamoDS/DynamoRevit/blob/7270187e8688fbd8ec538b4617810dccdb9a695f/src/Libraries/RevitNodes/Elements/TextNote.cs#L206

so you would build a default constructor for export options, it could be internal or hidden from the node library.